src/web/command_control/client/components/mission/RunPanel.tsx

```diff
@@ -20,9 +20,16 @@ type RunPanelProps = {
 type RunPanelState = {}
 
 export default class RunPanel extends React.Component<RunPanelProps, RunPanelState> {
-    addDuplicateRun(goals: Goal[]) {
-        Missions.addRunWithGoals(-1, goals, this.props.mission)
-    }
+    /**
+     * Executes the static function duplicateRun which copies
+     * the run parameter and creates a new run from the run that is
+     * passed as a parameter
```

Comment:
"Executes the static function duplicateRun which copies the run argument and creates a new run". I think this is all the text we need for this description